developerportal/apps/externalcontent/migrations/0023_auto_20190925_1338.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 2.2.4 on 2019-09-25 13:38
+
+from django.db import migrations, models
+import django_countries.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('externalcontent', '0022_auto_20190819_1304'),
+    ]
+
+    operations = [
```

Comment:
FYI this migration will drop the location and venue fields, and add city and country. However there is only one `externalevent` on the staging site so you could add this data back in manually once the migration has been run.

Reply:
Thanks mate, yeah I figured it'd be okay since the staging data is pretty minimal at the moment.